src/components/remotecontrol/remotecontrol.scss

```
@@ -393,19 +400,16 @@
     }
 
     .nowPlayingPageImage {
-        /* width: inherit; */
         overflow-y: hidden;
         overflow: hidden;
         margin: 0 auto;
     }
 
-    .nowPlayingPageImage.nowPlayingPageImageAudio {
-        width: 100%;
-    }
-
+    .nowPlayingPageImage.nowPlayingPageImageAudio,
     .nowPlayingPageImageContainer.nowPlayingPageImagePoster {
```

I believe this (#1056):
https://github.com/jellyfin/jellyfin-web/blob/68f7002ca82b7064c857c4631261d06b79551535/src/components/remotecontrol/remotecontrol.js#L251-L257
should be:
```js
        context.querySelector('.nowPlayingPageImage').classList.toggle('nowPlayingPageImageAudio', item.Type === 'Audio');
        context.querySelector('.nowPlayingPageImage').classList.toggle('nowPlayingPageImagePoster', item.Type !== 'Audio');
```

and 
```suggestion
    .nowPlayingPageImage.nowPlayingPageImagePoster {
```

and remove
https://github.com/jellyfin/jellyfin-web/blob/68f7002ca82b7064c857c4631261d06b79551535/src/components/remotecontrol/remotecontrol.scss#L415-L418

---

I believe these should be good now. :crossed_fingers: